### DISCUSSION TITLE
Help Needed - International Decimal Separator

### DISCUSSION THREAD

**Discussion by GCuser99** [GCuser99/SeleniumVBA#107] [General]
@6DiegoDiego9 recently made an improvement to SeleniumVBA's VBA source to properly handle International decimal separators in the WebJsonConverter class for use with Excel. I would like to do the same for the twinBASIC and for Access VBA sources, but without having to refer to Excel's Application object. Here is what works in Excel:
```
DecimalSep = Application.International(xlDecimalSeparator)
```
So I searched the VB6 forums and found the following simple code:
```
'https://www.vbforums.com/showthread.php?640664-RESOLVED-Decimal-separator
Sub TestDecimalSeparator()
    Dim num As Double
    num = 1000.1
    Debug.Print "My decimal separator is: " & DecimalSep()
    If DecimalSep <> "." Then
        Debug.Print "My number converted to String: " & Replace$(num, DecimalSep, ".")
    Else
        Debug.Print "My number converted to String: " & CStr(num)
    End If
End Sub

Function DecimalSep() As String
    DecimalSep = Format$(0, ".")
End Function
```
I tested and it gives the correct answers for my North American region, but I'm hoping others can test and report back if it works on their systems as well? Appreciate your help in advance!

Thanks, Mike

**Comment by 6DiegoDiego9** [comment]
Hi Mike! Oh sorry, I didn't know it doesn't work in twinBasic and for Access.
Anyway, it works for Italian too and it makes sense to me 👍
 ![image](https://github.com/user-attachments/assets/93cf9ca5-67df-4584-bab9-84ec47c8fabf)
